pixel.js

```diff
@@ -116,6 +137,10 @@ function setupCli() {
 		'-c, --change-id <Change-Id...>',
 		'The Change-Id to use. Use multiple flags to use multiple Change-Ids (e.g. -c <Change-Id> -c <Change-Id>)'
 	] );
+	const groupOpt = /** @type {const} */ ( [
+		'-g, --group <(mobile|desktop)>',
+		'The group of tests to run. If omitted the group will be desktop.'
```

Comment:
The default option (desktop) can be added as the third param which should make L90 unnecessary